### PR TITLE
wrote answers

### PR DESCRIPTION
**Pull request by Knguyen21**

### CODE REVIEW

File: lib/diagnostic.rb [hunk 1]
```diff
@@ -6,21 +6,21 @@
...
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = pry
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'star wars episode viii'
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: this will not work, need snake case
 Diagnostic.variable = 'star_wars_episode_viii'

File: lib/diagnostic.rb [hunk 2]
```diff
@@ -40,7 +40,8 @@
 def method_name
 end
 
-Diagnostic.defined_method = # call your method here
+Diagnostic.defined_method = (a, b).reduce { |a, b| (a + b) * 2}
+
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: Dont over think it
```
def add_and_multiply(a, b)
  2 * (a + b)
end
```

File: lib/diagnostic.rb [hunk 3]
```diff
@@ -49,31 +50,31 @@ def method_name
 
 # define your method here
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = odd?
```
Comment (J-Weeks) [inline lib/diagnostic.rb]:
```
def odd?(number)
  number % 2 == 1
end

Diagnostic.predicate = odd?(4)
```

File: lib/diagnostic.rb [hunk 4]
```diff
@@ -49,31 +50,31 @@ def method_name
...
 
 # Numbers in Ruby
 #
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = 'float'
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: caps, Float

**Comment by J-Weeks** [comment]
12.5/16.  Good job.  Don't over think it.  I feel that you grasp these concepts but sometimes get into your own head.

File: lib/diagnostic.rb (hunk 1 above)
Comment (Knguyen21) [inline lib/diagnostic.rb]: opps.. I thought you wanted to set the variable value to 'star wars episode viii' . I didn't read the question correctly

File: lib/diagnostic.rb (hunk 3 above)
Comment (Knguyen21) [inline lib/diagnostic.rb]: Okay.. I didn't read the question fully. I didn't know we were suppose to write a method out.

File: lib/diagnostic.rb (hunk 2 above)
Comment (Knguyen21) [inline lib/diagnostic.rb]: ohhh.. OKay.  Yeah, I'm definitely overthinking

**Pull request closed without merging**